website.config.ts

```diff
@@ -59,7 +59,7 @@ export const site = {
   title       : `${org.project.name} Docs`,
   tagline     : org.project.description,
   icon        : 'img/favicon.ico',
-  description : `Learn how to use ${org.project.name}.`,
+  description : `Learn to use the iR Engine.`,
```

Comment:
This shouldn't be hardcoded. It should still use a variable like before, that points to the correct desired name.

If `the` is desired, then it should be `the ${org.project.name}.`